book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, self, person)
+  end
+end
```

Comment:
- **NOTE: Kindly Make available and visible the missing files for better Review:**
# In Person file

![Capture](https://user-images.githubusercontent.com/63972670/194175931-b20ac78d-db43-4861-80b7-c03d9d950016.JPG)
- **@rentals should be initialized**: Add code like this `@rentals = []`
- **A method for adding rentals should be present**: Please [open here to see an example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)